src/irr/builtin/CMakeLists.txt

```diff
@@ -0,0 +1,95 @@
+function(irr_add_builtin_resources)
```

Comment:
again... no `irr` directory, no `irr` files.. did a merge go wrong or something?

Reply:
buggy and irrelevant, resolving